cli/providers/aws/elb/instances.go

```diff
@@ -0,0 +1,24 @@
+package elb
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elb"
+)
+
+//Instances returns the instance ids of the instances registered in the ELB
+func Instances(name string, config *aws.Config) []string {
+	instanceIDs := []string{}
+	svc := elb.New(session.New(), config)
+	dii := &elb.DescribeInstanceHealthInput{
+		LoadBalancerName: aws.String(name),
+	}
+	dio, err := svc.DescribeInstanceHealth(dii)
+	if err != nil {
+		return instanceIDs
```

Comment:
should we return an error along with the default value here so the caller knows that the function failed?

Reply:
The caller does not handle the error (there is nothing it can do), and I don't expect it to change.
I would keep it like this for now.